.github/ISSUE_TEMPLATE/story-issue-template-.md

```diff
@@ -8,26 +8,36 @@ assignees: ''
 
 # At a glance
 
-[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
+[comment]: # "Begin with a short summary so intent can be understood at a glance."
 
+[comment]: # "As: a person or role"
```

Comment:
I still think you should be going with the "In order to" template here [for all the reasons](https://blog.crisp.se/2014/09/25/david-evans/as-a-i-want-so-that-considered-harmful), but I'll leave it alone.